### PR TITLE
refactored bigquery

### PR DESCRIPTION
**Pull request by nacnudus**
- Remove statement about hours of operation
- Refactor SQL to simplify the frontend
- Reinstate the infobox for roles and persons

### CODE REVIEW

File: bigquery.ts [hunk 1]
```diff
@@ -85,135 +85,49 @@ const sendInitQuery: SendInitQuerySignature = async function() {
...
-      ;
-    `, { name }),
-    bigQuery(`
-      SELECT organisation.title
-      FROM graph.organisation
-      INNER JOIN graph.has_successor USING (url)
-      INNER JOIN graph.organisation AS successor ON has_successor.successor_url = successor.url
-      WHERE successor.title = @name
-    `, { name })
+  const [ bqOrganisation ] = await Promise.all([
```
Comment (maxf) [inline bigquery.ts:105]: If there's only one query we won't need `Promise.all`

File: bigquery.ts [hunk 2]
```diff
@@ -85,135 +85,49 @@ const sendInitQuery: SendInitQuerySignature = async function() {
...
-        taxon_levels.level AS level
-      FROM graph.taxon AS parent
-      INNER JOIN graph.has_parent ON has_parent.parent_url = parent.url
-      INNER JOIN graph.taxon AS child ON child.url = has_parent.url
-      INNER JOIN content.taxon_levels ON taxon_levels.url = child.url
-      WHERE parent.title = @name
-      ;
-    `, { name })
-  ]);
+  const [ bqTaxon ] = await Promise.all([bigQuery(
```
Comment (maxf) [inline bigquery.ts:88]: same as below: `Promise.all` no longer needed

**Review by maxf — APPROVED** [review]
brilliant!

**Comment by nacnudus** [comment]
Cypress passed.

**Pull request merged**